src/System.Net.HttpListener/src/System/Net/HttpListenerResponse.cs

```diff
@@ -27,20 +25,12 @@ public WebHeaderCollection Headers
             }
         }
 
+        public Encoding ContentEncoding { get; set; }
```

Comment:
But... I do see places in the managed implementation that assume this is never null, e.g. https://github.com/dotnet/corefx/blob/master/src/System.Net.HttpListener/src/System/Net/Managed/HttpConnection.cs#L453.  This change breaks that.